### DISCUSSION TITLE
Import from old system

### DISCUSSION THREAD

**Discussion by dnikola** [crater-invoice-inc/crater#576] [Ideas]
Hello

i would like to import invoices from existing system.

Can you tell me what api endpoint's to use for this? Any documentation on this?

Thanks

**Comment by clemsos** [comment]
agree, to write a basic script to import existing records from a csv file is what I lack now to migrate my accounting to crater

I started to draft something here : https://gist.github.com/clemsos/4398cee924d3056d1f5838f95bf0cb4a

This currently throws errors, as companies need to be created before invoices, then linked using their id I think.
